Description: Create samples directory via patch so it's available for autopkgtest
Author: Axel Beckert
Forwarded: not-needed

Index: libchart-perl/Makefile.PL
===================================================================
--- libchart-perl.orig/Makefile.PL	2014-09-22 15:20:13.000000000 +0200
+++ libchart-perl/Makefile.PL	2014-09-22 15:20:13.000000000 +0200
@@ -1,8 +1,5 @@
 use ExtUtils::MakeMaker;
 
-#  make the samples directory
-mkdir ('samples', 00755);
-
 #  write the makefile
 WriteMakefile ( 'NAME' => 'Chart',
 		'PREREQ_PM' => { 'GD' => 2.0 },
Index: libchart-perl/samples/.non-empty-directory
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ libchart-perl/samples/.non-empty-directory	2014-09-22 15:20:51.000000000 +0200
@@ -0,0 +1 @@
+needed to create the samples directory via patch for making autopkgtest to work properly
